slapp/transforms/video_utils.py

```diff
@@ -59,10 +59,16 @@ def transform_to_mp4(video: np.ndarray, output_path: str,
     Returns:
 
     """
+
+    # ffmpeg expects the video shape in width, height not row, col
+    # have to reverse shape when inputting
+    # gray8 is uint8 format
+
     writer = mpg.write_frames(output_path,
-                              video[0].shape,
-                              pix_fmt_in="gray",
-                              pix_fmt_out="gray",
+                              (video[0].shape[1], video[0].shape[0]),
+                              pix_fmt_in="gray8",
+                              pix_fmt_out="yuv420p",
+                              codec="libvpx-vp9",
                               fps=fps,
                               bitrate=bitrate)
```

Comment:
This is probably okay? But it does I think put us into constant bitrate mode. Constant quality encoding might be better? I'm not sure it's worth the extra effort though to dig through how imagio-ffmpeg is translating commands to ffmpeg.

See: https://trac.ffmpeg.org/wiki/Encode/VP9

Reply:
There's an option to add command line arguments but I think for times sake I'd rather pass on it. I'll create a ticket and throw it in the icebox.

Reply:
Don't put it in icebox please. New issues should go in "New Issues" and then be prioritized by Wayne during backlog grooming.

Reply:
I added the ticket to 'new issues', [ticket](https://app.zenhub.com/workspaces/allensdk-10-5c17f74db59cfb36f158db8c/issues/alleninstitute/segmentation-labeling-app/94)